test/helper.js

```diff
@@ -3,58 +3,42 @@ var data        = require('../hooks/data');
 var taskcluster = require('taskcluster-client');
 var taskcreator = require('../hooks/taskcreator');
 var v1          = require('../routes/v1');
-var bin = {
-  server:             require('../bin/server')
-};
+var load        = require('../bin/main');
+var config      = require('typed-env-config');
+var _           = require('lodash');
 
-var testProfile = 'test';
+var loadOptions = {profile: 'test', process: 'test-helper'};
+var cfg = config({profile: 'test'});
```

Comment:
> If I'm reading your middle comment correctly, I'm hearing you say that the elegant DI mock, which works perfectly well and isn't dependent on external services, should be replaced...

Nope, lets not replace it... 

Sorry, I was just saying how I normally would do
.. And yes, it would be ugly :)